pkg/testcase/upgradecluster.go

```diff
@@ -151,11 +149,12 @@ func getChannel() string {
 	if err != nil {
 		return err.Error()
 	}
+
 	var defaultChannel = fmt.Sprintf("INSTALL_%s_CHANNEL=%s", strings.ToUpper(product), "stable")
 
-	if customflag.ServiceFlag.InstallType.Channel != "" {
+	if customflag.ServiceFlag.Channel.Channel != "" {
```

Comment:
Can we have only one Channel option? `Channel.Channel` is confusing...

Reply:
done

Reply:
Has this been addressed? I don't see that it has been updated...

Reply:
i think u are looking at an outdated file right ?

Reply:
Looks good